sql/firefox_accounts_derived/fxa_log_content_other_events_v1/query.sql

```diff
@@ -0,0 +1,71 @@
+SELECT
+  jsonPayload.event,
+  jsonPayload.flow_time,
+  jsonPayload.locale,
+  jsonPayload.useragent,
+  jsonPayload.country,
+  jsonPayload.entrypoint,
+  jsonPayload.flow_id,
+  jsonPayload.region,
+  jsonPayload.service,
+  jsonPayload.utm_campaign,
+  jsonPayload.utm_content,
+  jsonPayload.utm_medium,
+  jsonPayload.utm_source,
+  -- timestamp, -- This is a static value in 1970 for all records
+  receiveTimestamp
```

Comment:
I might argue for renaming `receiveTimestamp` to `timestamp` here since I believe it's the value that logically _should_ be flowing to `timestamp`.

Reply:
I agree with that.